### PR TITLE
Add curating-event script and its tutorial

### PR DESCRIPTION
**Pull request by Benz-Poobua**
### python script
1. Rename the fields, Boolean correction, Time correction
2. Curate well-located events based on: 
                1. 6+ observations (P or S wave picks): nobs >= 6
                2. 4+ observing stations: nsta >= 4
                3. no fixed values (depth, epicenter, or time): ['fdepth', 'fepi', 'ftime'] are set to False
                4. a minimum observing station distance of no more than 10 km: distance <= 10
                5. horizontal and vertical errors of 10 km or less: erhor & sdep <= 10
                6. travel time RMS misfit of no more than 1 second: wrms <= 1
### notebook
- provide the tutorial on how to call the module and generate the output file (.csv) in `src/data/MtBaker_50km_radius_origins_curated`

### CODE REVIEW

File: .DS_Store
Comment (nts345045) [inline .DS_Store:1]: Please eliminate .DS_Store files (and similar auto-generated files/directories) from your repository.

File: src/.DS_Store
Comment (nts345045) [inline src/.DS_Store:1]: Please eliminate .DS_Store files (and similar auto-generated files/directories) from your repository.

File: src/notebooks/MtBaker_well_located.ipynb
Comment (nts345045) [inline src/notebooks/MtBaker_well_located.ipynb:1]: Please add some visualization of the subset origins. Particular parameters of interest are event type, location, origin time, magnitude, and parameters used to create the data subset. In addition, please provide figures comparing the distribution of each filtering parameter for the whole data set and the data subset.

Comment (nts345045) [reply src/notebooks/MtBaker_well_located.ipynb:1]: Finally, add a markdown block at the end of the notebook and write a set of observations for informative patterns you see in the data (note that not all data combinations may be particularly informative).

File: src/python/well_located_events.py
Comment (nts345045) [inline src/python/well_located_events.py:1]: Add a doctoring to the `convert_to_unix` method.

**Review by nts345045 — CHANGES_REQUESTED** [review]

**Comment by Benz-Poobua** [comment]
- Add docstring to the `convert_to_unix` method.
- Add subplots to display the well-located and whole datasets

File: data/Processed Datasets/MtBaker_50km_radius_origins_cleaned.csv
Comment (nts345045) [inline data/Processed Datasets/MtBaker_50km_radius_origins_cleaned.csv:1]: For cleanliness of the main repository, do not include intermediate output files as part of the repository.

File: data/Processed Datasets/MtBaker_50km_radius_origins_curated.csv
Comment (nts345045) [inline data/Processed Datasets/MtBaker_50km_radius_origins_curated.csv:1]: For cleanliness of the main repository, do not include intermediate output files as part of the repository.

File: data/Processed Datasets/MtBaker_50km_radius_origins_cleaned.csv
Comment (nts345045) [reply data/Processed Datasets/MtBaker_50km_radius_origins_cleaned.csv:1]: You can do the following:
add a `processed_data` directory to the root directory of your local repository
save intermediate data files in `processed_data`
add `processed_data` to your `.gitignore`

File: data/Processed Datasets/MtBaker_50km_radius_origins_curated.csv
Comment (nts345045) [reply data/Processed Datasets/MtBaker_50km_radius_origins_curated.csv:1]: You can do the following:
add a `processed_data` directory to the root directory of your local repository
save intermediate data files in `processed_data`
add `processed_data` to your `.gitignore`

**Review by nts345045 — CHANGES_REQUESTED** [review]
Code updates look good - conduct the intermediate data cleanup tasks for a merge approval.

**Review by nts345045 — APPROVED** [review]

**Pull request merged**